### PR TITLE
[Explicit Module Builds] Rely on swift-driver API to merge dependency graphs across targets

### PR DESCRIPTION
**Pull request by artemcm**
Builds on: https://github.com/apple/swift-driver/pull/284

### CODE REVIEW

**Comment by artemcm** [comment]
@swift-ci please smoke test

**Review by abertelrud — APPROVED** [review]
Looks good to me.  If this introduces a dependency on a new member function in swift-driver, do we need to wait for a swift-driver PR to land before merging this?

**Comment by artemcm** [comment]
> Looks good to me. If this introduces a dependency on a new member function in swift-driver, do we need to wait for a swift-driver PR to land before merging this?

Thank you. 
It relies on an API introduced in https://github.com/apple/swift-driver/pull/284, which is already merged so this should be good to go.

**Pull request merged**
